tools/go-agent/instrument/logger/frameworks/buildin.go

```diff
@@ -0,0 +1,42 @@
+package frameworks
+
+import (
+	"github.com/apache/skywalking-go/tools/go-agent/instrument/plugins/rewrite"
+	"github.com/dave/dst"
+	"github.com/dave/dst/dstutil"
+)
+
+type Buildin struct {
```

Comment:
Why you need to add this? It's not related to the grpc plugin.

Reply:
Sorry about this. I forgot the remove the unrelated lines.